src/cdk/lambda/node14func.ts

```diff
@@ -32,17 +34,19 @@ export class Node14Func extends NodejsFunction {
     let { keepNames, ...bundlingRest } = bundling
     if (typeof keepNames == "undefined") keepNames = true
 
-    const newProps = {
+    const newProps: NodejsFunctionProps = {
       ...rest,
       awsSdkConnectionReuse,
       environment,
       runtime,
       bundling: {
         keepNames,
-        bundlingRest,
+        ...bundlingRest,
       },
     }
 
     super(scope, id, newProps)
+
+    this.bundling = bundling
```

Comment:
Are we overriding here the default cdk construct for Lambdas? What does bundling do here?

Reply:
It's just informational, I only added it to verify it in the test. It doesn't have any function